eglot-fsharp.el

```diff
@@ -96,12 +97,12 @@
 					   nil "tool" "uninstall"
 					   "fsautocomplete" "--tool-path"
 					   default-directory))
-                (error  "'dotnet tool uninstall fsautocomplete --tool-path %s' failed" default-directory))))
-          (unless (zerop (call-process "dotnet" nil `(nil ,stderr-file) nil
-				       "tool" "install" "fsautocomplete"
-				       "--tool-path" default-directory "--version"
-				       version))
-            (error "'dotnet tool install fsautocomplete --tool-path %s --version %s' failed" default-directory  version)))
```

Comment:
This code block is always evaluated even if desired version is installed. I suppose if this expression should be evaluated if `(eglot-fsharp-current-version-p version)` is `nil`